Gemfile

```diff
@@ -11,15 +11,15 @@ gem 'jbuilder', '~> 2.5'
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 gem 'rails', '~> 5.2.2', '>= 5.2.2.1'
-gem 'webpacker-react'
+gem 'webpacker-react', '0.3.2'
 
 gem 'sassc-rails'
 gem 'turbolinks', '~> 5'
 gem 'uglifier', '>= 1.3.0'
-gem 'webpacker'
+gem 'webpacker', '4.0.0'
 gem 'bootstrap', '~> 4.3.1'
 
-gem "aws-sdk-s3", require: false
+# gem "aws-sdk-s3", require: false
```

Comment:
did you mean to remove amazon s3 gem, @Brantron ?

Reply:
yes, it was failing the build since we don't have creds